plugins/CoreConsole/Commands/CoreArchiver.php

```diff
@@ -34,32 +34,27 @@ public static function makeArchiver($url, InputInterface $input)
 
         $archiver->disableScheduledTasks = $input->getOption('disable-scheduled-tasks');
         $archiver->acceptInvalidSSLCertificate = $input->getOption("accept-invalid-ssl-certificate");
-        $archiver->shouldArchiveAllSites = (bool) $input->getOption("force-all-websites");
         $archiver->shouldStartProfiler = (bool) $input->getOption("xhprof");
         $archiver->shouldArchiveSpecifiedSites = self::getSitesListOption($input, "force-idsites");
         $archiver->shouldSkipSpecifiedSites = self::getSitesListOption($input, "skip-idsites");
-        $archiver->forceTimeoutPeriod = $input->getOption("force-timeout-for-periods");
-        $archiver->shouldArchiveAllPeriodsSince = $input->getOption("force-all-periods");
-        $archiver->restrictToDateRange = $input->getOption("force-date-range");
         $archiver->phpCliConfigurationOptions = $input->getOption("php-cli-options");
-
-        $restrictToPeriods = $input->getOption("force-periods");
```

Comment:
Ideally, if possible, be great to also add back force-periods

Reply:
That should also be supported... oh they're handled below. You must've missed that.

Reply:
Thanks, totally missed that one